src/Net/Soap/SoapOptions.php

```diff
@@ -95,6 +95,106 @@ public function getProxyPassword()
         return $this->get('proxy_password');
     }
 
+    public function setEncoding($encoding)
+    {
+        return $this->set('encoding', "$encoding");
+    }
+
+    public function getEncoding()
+    {
+        return $this->get('encoding');
+    }
+
+    public function setTrace($trace)
+    {
+        return $this->set('trace', (bool)$trace);
+    }
+
+    public function getTrace()
+    {
+        return $this->get('trace');
+    }
+
+    public function setExceptions($exceptions)
+    {
+        return $this->set('exceptions', (bool)$exceptions);
+    }
+
+    public function getExceptions()
+    {
+        return $this->get('exceptions');
+    }
+
+    public function setClassmap(array $classmap)
+    {
+        return $this->set('classmap', $classmap);
+    }
+
+    public function getClassmap()
+    {
+        return $this->get('classmap');
+    }
+
+    public function setConnectionTimeout($timeout)
+    {
+        return $this->set('connection_timeout', $timeout|0);
+    }
+
+    public function getConnectionTimeout()
+    {
+        return $this->get('connection_timeout');
+    }
+
+    public function setTypemap(array $typemap)
+    {
+        return $this->set('typemap', $typemap);
+    }
+
+    public function getTypemap()
+    {
+        return $this->get('typemap');
+    }
+
+    public function setCacheWsdl($cacheWsdl)
```

Comment:
I understand you're following the conventions in this file by not including any docblocks, however it is not a particularly good convention nor a deliberate one. I think having docblocks would be beneficial, especially for methods such as this one where it is unclear what values can be passed. It is fine to copy excerpts directly from the documentation. For example, we could write, *Sets the WSDL caching mode to one of WSDL_CACHE_NONE, WSDL_CACHE_DISK, WSDL_CACHE_MEMORY or WSDL_CACHE_BOTH*.